chainer_compiler/elichika/typing/types.py

```diff
@@ -349,14 +349,14 @@ def TyChainerVariable(dtype, ndim=None, shape=None):
 
 # ------------------------ TypeChecker internal types --------------------------
 
-counter = 0
+var_counter = 0
```

Comment:
FYI: Actually we don't really need this counter variable to distinguish different TyVars and I'm just keeping it for `show()`. If you think it should be removed, please feel free to do so.

Reply:
Yeah, I realized this is only for debugging. I personally like these kind of features so I'd like to keep this at least for a while. Let's remove this if it turns out this isn't not very useful for debugging.